e2-studio-star-rx72n-firmware/src/tasks/temp_sensor_task.c

```
@@ -1177,6 +1171,9 @@ static void internal_temp_task_entry(ULONG input)
   rx_log_info(s_tag, "Temperature sensing running @ 1 Hz");
 
   /* Main polling loop */
+  temp_sensor_state_t state;
+  float               temp_celsius;
```

Clang (14.0.6)

[warning] variable 'temp_celsius' is not initialized

(cppcoreguidelines-init-variables)